ps0.py

```diff
@@ -1,10 +1,14 @@
-# 1. Ask the user to enter a number "x"
-
-
-# 2. Ask the user to enter a number "y"
-
-
-# 3. Prints out the number "x" raised to the power "y"
-
-
-# 4. Prints out the log (base 2) of "x"
\ No newline at end of file
+#Import the module "math" to gain access to the basic mathematical functions
+import math
+
+#Asking the user for a  value to assign to "x"
+x = float(input("Enter a value for x: "))
+
+#Asking the user for a value to assign to "y"
+y = float(input("Enter a value for y: "))
+
+#Printing the value of "x" raised to the power of "y"
+print("\n" + ""x" raised to the power "y" = " + str(x ** y) + "\n")
+
+#Printing the value of log base 2 of "x"
+print("Log base 2 of "x" = " + str(math.log(x,2)) + "\n")
```

Comment:
There exists math.log2(x) which is usually more accurate, but this is still good work.